pkgs/development/tools/continuous-integration/buildbot/master.nix

```diff
@@ -156,7 +156,7 @@ let
       description = "An open-source continuous integration framework for automating software build, test, and release processes";
       homepage = "https://buildbot.net/";
       changelog = "https://github.com/buildbot/buildbot/releases/tag/v${version}";
-      maintainers = with maintainers; [ ryansydnor lopsided98 ];
```

Comment:
Dropping `ryansydnor`, they were added as a maintainer when buildbot was first added to nixpkgs but don't seem to have ever been active in nixpkgs.

Reply:
ping @ryansydnor just in case. We can also re-add them in the unlikely case.